melodic/distribution.yaml

```diff
@@ -15380,6 +15380,21 @@ repositories:
       url: https://github.com/clearpathrobotics/wireless.git
       version: master
     status: maintained
+  witmotion_ros:
+    doc:
+      type: git
+      url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
+      version: main
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
+      version: 0.11.18
```

Comment:
This definitely isn't going to work; the release section here needs to be filled out by bloom.  I'll suggest you remove this, and if you want to do a release follow the instructions at http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

Reply:
I will remove this section for now, then me and my colleagues consider involving bloom. Thanks for an advice! I will update the PR soon.